ur_robot_driver/doc/features.md

```diff
@@ -2,23 +2,25 @@
 
 | Feature                                               | ROS2 Driver
 | ---                                                   | ---                       |
-| position-based control                                | yes                       |
-| scaled position-based control                         | yes                       |
-| velocity-based control                                | no<sup>1</sup>            |
-| reporting of tcp wrench                               | not yet implemented       |
-| reporting of tcp wrench in tcp link                   | not yet implemented       |
+| joint-position-based control                          | yes                       |
+| scaled joint-position-based control                   | yes                       |
+| joint-velocity-based control                          | no<sup>1</sup>            |
+| Cartesian position-based control                      | no                        |
+| Cartesian twist-based control                         | no                        |
+| Trajectory forwarding for execution on robot          | no                        |
+| reporting of tcp wrench                               | yes                       |
 | pausing of programs                                   | yes                       |
 | continue trajectories after EM-Stop resume            | yes                       |
 | continue trajectories after protective stop           | yes                       |
 | panel interaction in between possible                 | yes                       |
 | get and set IO states                                 | yes                       |
-| use [tool communication forwarder](https://github.com/UniversalRobots/Universal_Robots_ToolComm_Forwarder_URCap) on e-series | should work, not yet tested with ROS2            |
+| use [tool communication forwarder](https://github.com/UniversalRobots/Universal_Robots_ToolComm_Forwarder_URCap) on e-series | yes            |
 | use the driver without a teach pendant necessary      | yes                       |
-| support of CB2 robots                                 | -                         |
+| support of CB2 robots                                 | no                        |
 | trajectory extrapolation on robot on missing packages | yes                       |
 | use ROS as drop-in for TP-programs                    | yes                       |
-| headless mode                                         | not yet implemented       |
-| extract calibration from robot                        | not yet implemented       |
+| headless mode                                         | yes                       |
+| extract calibration from robot                        | yes                       |
 | send custom script commands to robot                  | yes                       |
```

Comment:
@livanov93 What about sending script command to the robot. Is that supported in the current version?

Reply:
@urrsk It is not, I missed this. We don't have a string controller yet, so we only support string commands that are predefined (e.g. payload setting)

Reply:
Okay

Reply:
I have made an issue for the missing support of 'send custom script commands to robot' #251